demo/stun/test.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 ETH Zurich
```

Comment:
It would be good to also have a README.md for this demo. Possibly with some ASCII art like, e.g., in demo/file_transfer/README.md.